src/rf/apps/core/models.py

```diff
@@ -133,6 +133,10 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def has_copied_images(self):
+        image = self.layer_images.first()
```

Comment:
This was written under the assumption that a layer only has one kind. I can change it so we can support mixed kinds, although I doubt it will be useful.